### ISSUE TITLE
interactive job submition failed

### ISSUE THREAD

**Issue by JianguoRao** [icatproject/topcat_ijp_plugin#36]
This is critical without it I can't do any interactive job test!!!!!!!!!

I have tried to submit job "Select Polygon Region from Quincy Output NoOptions(PI)"
and got the following error:
--------------------
Job Submission

One or more job submissions failed:
Input Entity Ids 	Error response
418860 	No error response
Configure Job
Job Type
There are no configuration options for this job type
-------------------

check the ijp log file, it has:

2016-12-02 09:36:42,080 [http-listener-2(23)] INFO  BatchJson - Processing: class org.icatproject.ijp.batch.exceptions.InternalException code 1: Host key verification failed.
lost connection

2016-12-02 09:37:00,038 [__ejb-thread-pool8] ERROR MachineEJB - Unexpected problem using ssh to connect to vm77.nubes.stfc.ac.uk to find proceeses for pool451

Obviously, a communication broken down between ijp server and the worker node vm77

**Comment by brianritchie1312** [comment]
The reported error message appears to have happened because vm190 (IJP server) does not trust vm77's certificate. This is odd because I thought I had added the key to the dmf account's known_hosts. Did so again; but this leads to another error, "Authentication token manipulation error". This may be more serious.

For interactive jobs, the IJP creates a temporary user and assigns them a very short random password. This password is then shown to the user who has to type it into the remote desktop connection dialog; so it's important that it should not be too long or complicated. After a few minutes, the password is removed, so no-one can use it to log into that account; and the account itself is deleted once the session has been closed. The error message above comes from the attempt to set the initial password. My first guess was that vm77 is implementing some new draconian security policy that forbids short passwords; but when experimenting by hand I've been unable to set any password at all.

I will ask the Cloud service if they know of any reasons why the mechanism that has worked on every previous worker node (Cloud and otherwise) should have stopped working.
